sakuya/mewo.py

```diff
@@ -0,0 +1,123 @@
+from dataclasses import dataclass
+import random
+from typing import Dict
+
+from discord import Guild
+from discord.ext import commands
+from discord.ext.commands import Bot
+
+from .db import db, Guild as _Guild, Member as _Member
+
+
+random_mewo = [
+    "ME.. oh you're not mcmower.",
+    "MEWO",
+    "You didn't really expect me to say mewo did you"
+]
+
+
+@dataclass
+class GuildState:
+    guild: Guild
+    mewo_enabled: bool
+
+
+class Mewo(commands.Cog):
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.guilds: Dict[Guild, GuildState] = dict()
+        self.data_loaded = False
+
+    @commands.Cog.listener()
+    async def on_ready(self):
+        # This event fires on reconnects, but we only want it to run once
+        if not self.data_loaded:
+            self.data_loaded = True
+            self.load_from_db()
+            print('Mewo response ready.')
+
+    def load_from_db(self):
+        for g in db.query(_Guild).filter(_Guild.mewo_enabled.isnot(None)).all():
+            guild: Guild = self.bot.get_guild(g.id)
+            if guild:
+                if g.mewo_enabled:
+                    self.guilds[guild] = GuildState(guild=guild, mewo_enabled=True)
+                else:
+                    print(f"Mewo is not enabled on {guild.name}! Mewo responses disabled in guild.")
+            else:
+                print(f"Guild {g.id} not found during Mewo init.")
+
+    @commands.command()
+    async def mewo(self, ctx):
+        state = self.guilds.get(ctx.guild)
+        if state:
+
+            memberid = ctx.message.mentions[0].id if len(ctx.message.mentions) == 1 else ctx.author.id
+
+            member = db.query(_Member).get((memberid, ctx.guild.id)) or _Member(user_id=memberid,
+                                                                                guild_id=ctx.guild.id)
+
+            if member.mewo_text:
+                msg = member.mewo_text
+            else:
+                msg = random.choice(random_mewo)
```

Comment:
nit: `or` would work here. However, it's arguably less readable...

@Trif4 What do you think?

Reply:
Ah, this is leftover from when this was a slightly longer and more complicated `if`. Updated for now, can always be changed back if the slightly higher readability is wanted